catalog/ui/search-ui/standard/src/main/webapp/js/view/preferences/PreferencesModal.view.js

```diff
@@ -270,13 +270,13 @@ define([
             },
             resetDefaults: function () {
                 this.onEdit();
-                var model = this.model;
                 this.viewMapLayers.each(function (viewLayer) {
                     var url = viewLayer.get('url');
-                    var defaultConfig = model.getMapLayerConfig(url);
-                    viewLayer.set('show', defaultConfig.show);
+                    var defaultConfig = _.find(properties.imageryProviders, function (layerObj) {
```

Comment:
It's possible for this to be undefined if it can't find a match.  Not sure if it'll ever happen, but if so the ```viewLayer.set``` using ```defaultConfig.alpha``` will fail.